README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are empowered to call out and attempt to resolve disrespectful behavior, and admins are empowered to enforce the Code of Conduct.
+
+## The Short Version
+
+Be respectful of other people; respectfully call out anyone behaving inappropriately, ask them to stop, and direct them to this Code of Conduct; and if you need help resolving an issue, contact the admins. If you are unable to follow our Community Code of Conduct you will be removed from HangOps.
+
+## The Long Version
+
+### Respect
+
+HangOps is a community of people who care about running production systems reliably. We have diverse backgrounds, skills, talents, experiences, cultures, and opinions. We are an inclusive environment and we treat all individuals respectfully.
+
+We value respectful behavior above individual opinions.
+
+Respectful behavior includes:
+
+* Be considerate, kind, constructive, and helpful.
+* Avoid demeaning, discriminatory, harassing, hateful, or physically threatening behavior, speech, and imagery.
+* If you’re not sure, ask someone instead of assuming. No, really. Just ask the admins.
+* Model respectful behaviour, but also address disrespect when you see it. One of the best ways to create a respectful community is to speak up when you see disrespectful behavior—and not just rely on admins to act!
+
+### Resolve Peacefully
+
+If you see someone behaving disrespectfully, we urge you to respectfully dissuade them from such behavior.
```

Comment:
@jordansissel that's a valid concern, although I have questions about its implementation too.

Question 1) Should it happen that all of that occurs (abuse -> follow -> blocked/ejected abuser), but later atonement/correction occurs too (presuming months/years, not hours), what happens? Dealt with on a case-by-case basis?

Question 2) Scenario: we have 2+ people, all regular participants on Hangops (albeit in different channels/times/whatever), and some issues happen that affects one of these people to the degree of it being considered abuse - either by them, or against them. How is this dealt with?